src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -646,6 +641,35 @@ public static class AkkaClusterHostingExtensions
             builder.AddHocon(DistributedData.DistributedData.DefaultConfig(), HoconAddMode.Append);
             return builder;
         }
+
+        /// <summary>
+        ///     Applies a default setting that, if not overriden, will be applied to all shard regions
+        /// </summary>
+        /// <param name="builder">
+        ///     The builder instance being configured.
+        /// </param>
+        /// <param name="shardOptions">
+        ///     The set of options for configuring <see cref="ClusterShardingSettings"/>
+        /// </param>
+        /// <returns>
+        ///     The same <see cref="AkkaConfigurationBuilder"/> instance originally passed in.
+        /// </returns>
+        public static AkkaConfigurationBuilder WithDefaultClusterShardingSettings(
```

Comment:
Delete this method - don't want it. Each `ShardRegion` should have its own configuration.